### PR TITLE
3.6.0 release notes 

### PR DESCRIPTION
**Pull request by victoriaatraft**
## Summary of Changes
Delivers HTML updates for https://github.com/raft-tech/TANF-app/issues/3156

Testable at:
https://reitermb.github.io/Test/knowledge-center/

Screenshot 2024-09-09 at 2 22 22 PM
cc: @ttran-hub

### CODE REVIEW

File: product-updates/knowledge-center/index.html [hunk 1]
```diff
@@ -335,13 +335,71 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+        <ul class="usa-icon-list">
+          <li class="usa-icon-list__item" style="margin-top: 16px;">
+            <div class="usa-icon-list__icon text-ink">
+              <img src="../img/added.svg" aria-hidden="true" alt="" style="background-color: #d3ffcc; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+            </div>
+            <div class="usa-icon-list__content">
+              <h5 class="usa-icon-list__title">
+                Additional Context
+              </h5>
+              <p>Error messages regarding <a href="https://tdp-project-updates.app.cloud.gov/knowledge-center/viewing-error-reports.html"> values across related records</a> now include the associated item number(s), item name(s), and row number to help you identify records that have been detected as having data quality discrepancies. </p>
```
Comment (ADPennington) [inline product-updates/knowledge-center/index.html]: @victoriaatraft @reitermb can we add id attributes to the relevant error type descriptions for users to navigate directly to the relevant information in the KC? The following subheadings on the error report page would need id attributes:

- Errors related to inconsistent values for related items/data elements in the same record:
- Errors related to inconsistent values across related records

Comment (victoriaatraft) [reply product-updates/knowledge-center/index.html]: Sorry about that @ADPennington you provided the links in the gitbook and copy and pasting from the working file had reset the link and I missed.

File: product-updates/knowledge-center/index.html [hunk 2]
```diff
@@ -335,13 +335,71 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+    <li class="usa-icon-list__item" style="margin-top: 16px;">
+      <div class="usa-icon-list__icon text-ink">
+        <img src="../img/in-dev.svg" aria-hidden="true" alt="" style="background-color: #fff4bd; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+      </div>
+      <div class="usa-icon-list__content">
+        <h5 class="usa-icon-list__title">
+          Data Submission History File Status 
+        </h5>
+        <p>
+          We've received feedback about the "accepted with errors" and "rejected" statuses in the Submission History Table for active and aggregate case data and understand that this is confusing. We are currently working on clarifying the language and improving this experience with our partners to better meet your needs. <a href="https://tdp-project-updates.app.cloud.gov/knowledge-center/view-submission-history.html">Read more about submission history and file status.</a></p>
```
Comment (lfrohlich) [inline product-updates/knowledge-center/index.html]:
```suggestion
          We've received some requests for clarification around
[file submission status](https://tdp-project-updates.app.cloud.gov/knowledge-center/view-submission-history.html#:~:text=in%20the%20file.-,File%20Statuses%20Key,-Status)
and
[error messages](https://tdp-project-updates.app.cloud.gov/knowledge-center/viewing-error-reports.html#overview-of-the-error-report)
. In response, we're working to improve guidance and the user experience to better meet partners' needs. Additionally, the TDP team is planning more opportunities to connect with partners to discuss the TDP error report including terminology, what errors mean, etc. Please stay tuned.</a></p>
```

File: product-updates/knowledge-center/index.html [hunk 3]
```diff
@@ -335,13 +335,71 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+        <h4 style="margin-top: 16px; color: #595959;">In Development:</h4>
+        <ul class="usa-icon-list">
+  
+    <li class="usa-icon-list__item" style="margin-top: 16px;">
+      <div class="usa-icon-list__icon text-ink">
+        <img src="../img/in-dev.svg" aria-hidden="true" alt="" style="background-color: #fff4bd; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+      </div>
+      <div class="usa-icon-list__content">
+        <h5 class="usa-icon-list__title">
+          Data Submission History File Status
```
Comment (lfrohlich) [inline product-updates/knowledge-center/index.html]:
```suggestion
          File Status and Error Messaging 
```

File: product-updates/knowledge-center/index.html [hunk 4]
```diff
@@ -335,13 +335,75 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+      <div class="usa-icon-list__icon text-ink">
+        <img src="../img/in-dev.svg" aria-hidden="true" alt="" style="background-color: #fff4bd; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+      </div>
+      <div class="usa-icon-list__content">
+        <h5 class="usa-icon-list__title">
+          Data Submission History File Status 
+        </h5>
+        <p>
+          We've received some requests for clarification around
+[file submission status](https://tdp-project-updates.app.cloud.gov/knowledge-center/view-submission-history.html#:~:text=in%20the%20file.-,File%20Statuses%20Key,-Status)
```
Comment (ADPennington) [inline product-updates/knowledge-center/index.html]: this link

File: product-updates/knowledge-center/index.html [hunk 5]
```diff
@@ -335,13 +335,75 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+and
+[error messages](https://tdp-project-updates.app.cloud.gov/knowledge-center/viewing-error-reports.html#overview-of-the-error-report)
```
Comment (ADPennington) [inline product-updates/knowledge-center/index.html]: and this link

**Review by reitermb — APPROVED** [review]
Confirming all changes are in good shape, re-ran a11y scans on all three touched pages, all in the clear 🚀

**Review by ADPennington — APPROVED** [review]
LGTM! 🚀 cc: @ttran-hub

**Pull request merged**